### PR TITLE
Uplading my branch

### PR DESCRIPTION
**Pull request by diannasb67**

### CODE REVIEW

**Review by diannasb67 — COMMENTED** [review]
reviewed document

**Review by diannasb67 — COMMENTED** [review]
reviewed changes

File: index.html [hunk 1]
```diff
@@ -8,15 +8,12 @@
     <!-- Bootstrap CDN -->
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
         crossorigin="anonymous" />
-
```
Comment (diannasb67) [inline index.html:11]: removed line

**Review by diannasb67 — COMMENTED** [review]
approved changes

**Review by danielhomann4 — APPROVED** [review]

**Pull request merged**